### DISCUSSION TITLE
screen capture using pipewire

### DISCUSSION THREAD

**Discussion by moilong** [YaLTeR/niri#346] [Q&A] [answered]
wf-recorder gives this error:
`wl_registry@2: error 0: invalid version for global zwlr_screencopy_manager_v1 (25): have 1, wanted 3`
No pipewire source in obs.
xdg-dessktop-portal-gnome, xdg-desktop-portal-gtk are running.
Other compositors like labwc,swayfx and hyprland uses xdg-desktop-portal-wlr so wf-recorder and obs works.
Can't figure out the missing component for niri.
niri 0.1.5 (v0.1.5-106-g229ca90) on voidlinux.

**Comment by YaLTeR** [comment]
> wf-recorder

It doesn't use pipewire, it uses wlr-screencopy. niri doesn't do wlr-screencopy capture yet.

> No pipewire source in obs.

Should be there. If it's void linux, make sure you start pipewire before you start niri.

**Reply by moilong** [reply]
Starting pipewire with spawn-at -startup command. Do i need to start before that.

**Reply by YaLTeR** [reply, marked as answer]
Yes, it must be running before niri is started.
